scripts/build-color-variants.js

```diff
@@ -8,43 +8,59 @@ const defaultVariables = require('../src/variables');
 
 const allColors = [
   'gray-dark',
+  'gray-deep',
   'gray',
   'gray-light',
+  'gray-lighter',
   'gray-faint',
 
   'pink-dark',
+  'pink-deep',
   'pink',
   'pink-light',
+  'pink-lighter',
   'pink-faint',
 
   'red-dark',
+  'red-deep',
   'red',
   'red-light',
+  'red-lighter',
   'red-faint',
 
   'orange-dark',
+  'orange-deep',
   'orange',
   'orange-light',
+  'orange-lighter',
   'orange-faint',
 
   'yellow-dark',
+  'yellow-deep',
   'yellow',
   'yellow-light',
+  'yellow-lighter',
   'yellow-faint',
 
   'green-dark',
+  'green-deep',
   'green',
   'green-light',
+  'green-lighter',
   'green-faint',
 
   'blue-dark',
+  'blue-deep',
   'blue',
   'blue-light',
+  'blue-lighter',
   'blue-faint',
 
   'purple-dark',
+  'purple-deep',
   'purple',
   'purple-light',
+  'purple-lighter',
```

Comment:
I think we're going to want to exclude deep and lighter from all form color variant building (only make these new colors available through  `color-`, `bg-`, and `link-`).

Reply:
@samanpwbb Should this say `color-` `bg-` and `border-`? Link is currently included in forms based on the existing functions. I just updated to remove buttons as well, so we should see just these three.

Also, the function name `isNotAccessibleExceptButtons` is kind of confusing since if it returns true, we don't make a button variant. Should it be named like the other functions, maybe `isNotAccessibleForButtons` or am I misunderstanding?

Reply:
You are right! Agree with everything you're saying. Some of this conditional code is confusing, definitely. Looks like you've decoded it and therefore you should feel good about renaming it!